src/PortfolioGenerator.java

```diff
@@ -1,5 +1,85 @@
-public class PortfolioGenerator {
-    public static void main(String[] args) {
-        // Write your code here ... ⬇
+
+
+    import java.io.FileNotFoundException;
+    import java.util.Scanner;
+    import java.io.PrintWriter;
+    import java.util.ArrayList;
+    import java.io.File;
+
+
+
+public class PortfolioGenerator
+{
+    ;
+
+    public static void main(String[] args)
+    {
+        //Declair Variables
+        String name = "";
+        String email = "";
+        String background = "";
+        String nameFormatted = "";
+        String fileName = "fileoutput.txt";
+        ArrayList<String> pLang = new ArrayList<>();
```

Comment:
```suggestion
        ArrayList<String> programmingLanguages = new ArrayList<>();
```
would be a better name.
Avoid mental mapping when picking your variable names